x-pack/plugins/profiling/common/topn.ts

```diff
@@ -85,18 +88,35 @@ export function groupSamplesByCategory(samples: TopNSample[]): TopNSubchart[] {
     const series = seriesByCategory.get(sample.Category)!;
     series.push({ Timestamp: sample.Timestamp, Count: sample.Count });
 
-    total += sample.Count;
+    total += sample.Count ?? 0;
```

Comment:
How can a sample end up without a `Count` field ? And if it does, shouldn't the default be `1` instead of `0` ?

Reply:
(Same for line L97)

Reply:
A sample doesn't end up without a `Count` field, but according to the ES spec/types, the aggregation where `Count` gets its value from, can either be `number` or `null`. I think in this case that's theory only, so we can also get by by using `total += sample.Count!` (otherwise TypeScript will complain), but I think `?? 0` is the safer option here, and just as readable.